README.md

```diff
@@ -285,22 +272,32 @@ You can adjust the number of replicas or scale to a larger pod size (specified w
 ```typescript
 import { Pinecone } from '@pinecone-database/pinecone';
 const pc = new Pinecone();
-
-await pc.configureIndex('pod-index', { replicas: 3 });
+await pc.configureIndex('pod-index', {
```

Comment:
Note: diff `configureIndex` syntax btwn `2.2.2` and this release

Reply:
Thanks! I missed this as well. 🤦‍♂️